models/lido/liquidity/arbitrum/lido_liquidity_arbitrum_wombat_pools.sql

```diff
@@ -0,0 +1,182 @@
+                                "project",
+                                "lido_liquidity",
+                                \'["ppclunghe"]\') }}'
+    )
+}}
+
+{% set project_start_date =  '2023-05-25'%} 
+
+with dates as (
+    with day_seq as (select (sequence(cast('{{ project_start_date }}' as date), cast(now() as date), interval '1' day)) as day)
+select days.day
+from day_seq
+cross join unnest(day) as days(day)
+  )
+
+, tokens_prices_daily AS (
+    SELECT DISTINCT
+      DATE_TRUNC('day', minute) AS time,
+      contract_address as token,
+      AVG(price) AS price
+    FROM {{source('prices','usd')}}
+    WHERE
+      DATE_TRUNC('day', minute) >= timestamp '{{ project_start_date }}'
+      AND DATE_TRUNC('day', minute) < DATE_TRUNC('day', now())
+      AND blockchain = 'arbitrum'
+      AND contract_address = 0x5979d7b546e38e414f7e9822514be443a4800529
+    GROUP BY 1,2
+    UNION ALL
+    SELECT DISTINCT 
+      DATE_TRUNC('day', minute),
+      contract_address AS token,
+      LAST_VALUE(price) OVER (PARTITION BY  DATE_TRUNC('day', minute), contract_address  ORDER BY minute NULLS FIRST range BETWEEN UNBOUNDED preceding AND UNBOUNDED following) AS price
+    FROM {{source('prices','usd')}}
+    WHERE
+      DATE_TRUNC('day', minute) = DATE_TRUNC('day', now())
+      AND blockchain = 'arbitrum'
+      AND contract_address = 0x5979d7b546e38e414f7e9822514be443a4800529
+  )
+
+, wsteth_prices_hourly AS (
+SELECT
+      time,
+      LEAD(time,1,DATE_TRUNC('hour', now() + INTERVAL '1' hour)) OVER (ORDER BY time NULLS FIRST) AS next_time,
+      price
+    FROM
+      (
+    SELECT DISTINCT
+      DATE_TRUNC('hour', minute) AS time,
+      AVG(price) AS price
+    FROM {{source('prices','usd')}}
+    WHERE
+      DATE_TRUNC('day', minute) >= cast('{{ project_start_date }}' as timestamp)
+      AND DATE_TRUNC('day', minute) < DATE_TRUNC('day', now())
+      AND blockchain = 'arbitrum'
+      AND contract_address = 0x5979d7b546e38e414f7e9822514be443a4800529
+    GROUP BY 1
+  
+  ))
+
+, swap_events AS (
+    SELECT
+      DATE_TRUNC('day', sw.evt_block_time) AS time,
+      sw.contract_address AS pool,
+      SUM(case when fromToken = 0x5979d7b546e38e414f7e9822514be443a4800529 then cast(fromAmount as double) else -cast(toAmount AS DOUBLE) end) AS amount0,
+      SUM(case when fromToken = 0x5979d7b546e38e414f7e9822514be443a4800529 then -cast(toAmount as double) else cast(fromAmount AS DOUBLE) end) AS amount1
+    FROM {{source('wombat_arbitrum','wsteth_pool_evt_Swap')}} AS sw
+    WHERE DATE_TRUNC('day', sw.evt_block_time) >= timestamp '{{ project_start_date }}'
+    GROUP BY  1,2
+ )
+ 
+  , deposit_wsteth_events AS (
+    SELECT
+      DATE_TRUNC('day', sw.evt_block_time) AS time,
+      sw.contract_address AS pool,
+      SUM(cast(amount as double)) AS amount0
+    FROM {{source('wombat_arbitrum','wsteth_pool_evt_Deposit')}} AS sw
+    WHERE DATE_TRUNC('day', sw.evt_block_time) >= timestamp '{{ project_start_date }}'
```

Comment:
Same as above, you may retry with all `timestamp` replaced by `DATE` first.
```suggestion
    WHERE DATE_TRUNC('day', sw.evt_block_time) >= DATE '{{ project_start_date }}'
```